main.go

```diff
@@ -55,6 +66,13 @@ var containerToDuration = map[string]float64{}
 var containerDurations = []float64{}
 var containerDurationsMap = map[float64]struct{}{}
 
+const (
+	OutputTypeText string = "text"
+	OutputTypeCsv  string = "csv"
+)
+
+var outputType string = OutputTypeText
```

Comment:
go ahead and create a

var (
  ...
) 

section with your new variable plus the existing `containerOnly`